lib/pbench/test/unit/server/test_file_tree.py

```diff
@@ -188,20 +188,19 @@ def test_unpack(
         """Test to check the unpacking functionality of the FileTree"""
         source_tarball, source_md5, md5 = tarball
 
-        def mock_run(args, cwd, stdin, stderr, stdout, universal_newlines):
+        def mock_run(args, **kwargs):
             tarball_name = source_tarball.name[0 : -len(".tar.xz")]
             assert args == [
                 "tar",
                 "-xf",
                 (server_config.ARCHIVE / "ABC" / source_tarball.name),
             ]
-            assert cwd == server_config.INCOMING / "ABC" / tarball_name
-            assert stdin == subprocess.DEVNULL
-            assert stderr == subprocess.STDOUT
-            assert stdout == subprocess.PIPE
-            assert universal_newlines
+            assert kwargs["cwd"] == server_config.INCOMING / "ABC" / tarball_name
+            assert kwargs["stdin"] == subprocess.DEVNULL
+            assert kwargs["capture_output"]
+            assert kwargs["text"]
```

Comment:
I just wanted to confirm their behaviour in one of the test, that was the only reason I added them and it is for the same reason I did not include them in other tests.

Reply:
It's a valid point. Since we expect the exception to include the `stderr` we have some external vested interest that output is captured and passed-through in text format rather than as a binary string: but those assumptions are captured by testing the actual text of the exception in `test_unpack_with_returncode` so this is redundant... and also perhaps overly "white box" as there may be equally valid alternatives to achieve that.